src/kvlt/util.cljc

```diff
@@ -56,16 +56,17 @@
 (defn pprint-str [x]
   (str/trimr (with-out-str (pprint/pprint x))))
 
+(defn examples->str [examples]
+  (str "\n\n```clojure\n"
+       (str/join
+         "\n\n"
+         (for [[before after] examples]
+           (cond-> (pprint-str before)
+                   after (str "\n  =>\n" (pprint-str after)))))
+       "\n```"))
+
 (defn doc-examples! [vvar examples]
```

Comment:
This function is now unused internally to kvlt.  I don't know why any downstream consumer would use this, but I didn't want to break them.